docker/Dockerfile.debian.testing

```diff
@@ -57,7 +57,10 @@ RUN apt-get update && apt-get -y install \
   softhsm2 \
   sqlite3 \
   valgrind \
-  wget
+  wget \
+  doxygen \
+  graphviz
```

Comment:
Would you mind inserting these in the alphabetically-appropriate places? For comparison purposes, it makes it much easier to keep this list sorted!

Reply:
Sure, thanks! That makes sense.

Reply:
I just realized this is still an issue! Do you mind fixing this?

Reply:
I fixed this issue on the zee_test branch. Maybe I should just simply close this pull request?

Reply:
Sure, that's fine! So for future reference, see PR #1223.